internal/command/command.go

```diff
@@ -676,11 +677,14 @@ func ChangeWorkingDirectory(ctx context.Context, wd string) (context.Context, er
 }
 
 func createInvocationID(ctx context.Context) (context.Context, error) {
-	command := FromContext(ctx).CommandPath()
+	var (
+		command = FromContext(ctx).CommandPath()
+		guid    = xid.New()
+	)
 
-	invocationID := fmt.Sprintf("%s-%d", strings.ReplaceAll(command, " ", "-"), time.Now().UnixNano())
+	invocationID := fmt.Sprintf("%s-%s", strings.ReplaceAll(command, " ", "-"), guid.String())
```

Comment:
why keeping the command path?

Reply:
@dangra no particular reason, but I thought looking at it in honeycomb would be helpful when trying to narrow down the offending command.

Reply:
Is there a size limit for the honeycomb ID? the nice thing about xid is the small and fixed size IMO.
things like the command name could be sent using a different header, something that can be expanded to other debug data in the future. just saying.